### PR TITLE
Improve coords.py

### PR DESCRIPTION
**Pull request by nealwu**
Fixes https://github.com/tensorflow/minigo/issues/7

Also fixed a bucket name discrepancy in README.md.

Passes unit tests, and local_rl_loop.py seems to train normally.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,11 @@ pip-selfcheck.json
 sgf
 pyvenv.cfg
 .DS_store
+logs/
+
+# virtualenv folder
+minigo/
```
Comment (brilee) [inline .gitignore]: Interesting, my virtualenv got installed in ./bin/ and other misc. files. And I think Andrew's gets installed in ~/.virtualenvs/minigo. I'm starting to wonder how many virtualenv/venv/pipenv variants there are out there...

Comment (nealwu) [reply .gitignore]: Haha I see. I followed [this link](http://docs.python-guide.org/en/latest/dev/virtualenvs/#lower-level-virtualenv) from the README which just tells you to run `virtualenv my_project`, so naturally I named it `minigo`.

**Comment by brilee** [comment]
Ah, I see. In this case, I think it's appropriate to put it in a personal
.gitignore rather than the repo-specific gitignore? Since other people may
not have implemented their virtualenvs the same way.

**Comment by nealwu** [comment]
Sure that's fine; took it out.

**Pull request merged**